RELEASE/scripts/autoscend/iotms/clan.ash

```diff
@@ -49,7 +49,7 @@ boolean handleFaxMonster(monster enemy, boolean fightIt, string option)
 		return false;
 	}
 	// don't try to fax unfaxable monsters
-	if($monsters[smut orc pervert, screambat, War Frat 151st Infantryman] contains enemy)
+	if(!can_faxbot(enemy))
```

Comment:
`ash can_faxbot($monster[screambat])` returns true so something is wrong with it

Reply:
easyfax has screambat, you can fax it:
```
> ash faxbot($monster[screambat])

Configuring faxable monsters.
Configuring Easyfax (2504737)
Configuring CheeseFax (3038166)
Faxable monster lists fetched.
Visiting Fax Machine in clan VIP lounge
Asking Easyfax to send a fax of screambat: screambat
No response from Easyfax after 60 seconds.
Returned: false

> ash import autoscend; checkFax($monster[screambat])

Receiving a fax.
You acquire an item: photocopied monster
Preference photocopyMonster changed from to screambat
You receive a photocopied screambat from the fax machine.
Returned: true
 
```

Reply:
Huh the more you know. I thought it wasn't faxable for some reason. Good update!